### PR TITLE
add sorting on redux state for recent_edits via sortByKeys

### LINKED ISSUE #1638: Recent Edits sorting is wonky
Sorting is somehow implemented for the Recent Edits table (`/recent-activity/recent-edits`), probably via the List component. However, it doesn't work that well. Sorting by date does not work, and all the other sorting only works ascending; clicking again does not reverse the sorting like it does in some other tables.

Figuring how how to implement sorting nicely for redux-backed tables, starting with this one, would be a welcome improvement.

### PR DESCRIPTION
**Pull request by psinghal20**
Fixes #1638. Adds sorting on the redux state for recent_edits via using sortByKeys from model_utils.

### CODE REVIEW

**Comment by psinghal20** [comment]
@ragesoss The feature specs for language switcher seems to fail in Travis Build and I don't think it is related to this and must be random failing spec.

**Comment by ragesoss** [comment]
Yes, that one is flaky. I'll restart the build.

**Comment by ragesoss** [comment]
I'm trying this out, and sorting by date still is weird; I have to click twice before it will sort by date. Also, there are no icons in the header to indicate which columns are sortable and which are not.

File: app/assets/javascripts/components/activity/activity_table.jsx [hunk 1]
```diff
@@ -121,7 +108,7 @@ const ActivityTable = createReactClass({
   _renderHeaders() {
     return this.props.headers.map((header) => {
       return (
-        <th style={header.style || {}} key={header.key} onClick={this.sortItems} className="sortable" data-sort-key={header.key}>
+        <th style={header.style || {}} key={header.key} onClick={this.sortItems} className="sortable asc" data-sort-key={header.key}>
```
Comment (ragesoss) [inline app/assets/javascripts/components/activity/activity_table.jsx]: Ideally, this should be done the same way as the `/explore` table: the class changes according to the direction it's being sorted, so the sorting indicator shows which way that column is being sorted. I'll open a separate issue for this.

Comment (psinghal20) [reply app/assets/javascripts/components/activity/activity_table.jsx]: @ragesoss Okay, I understand. I can also work on that issue if you want

**Pull request merged**